src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1 +1,19 @@
-export const CommentInfo = () => <>Put the comment here</>;
+export const CommentInfo = ({ comment }) => {
+  const { name, email, body } = comment;
+
+  return (
+    <div className="CommentInfo">
+      <div className="CommentInfo__title">
+        <strong className="CommentInfo__name">{name}</strong>
+
+        {' by '}
```

Comment:
you can wrap with `span`
```suggestion
        <span> by </span>
```